.github/ISSUE_TEMPLATE/app-release.md

```diff
@@ -0,0 +1,33 @@
+---
+name: App release
+about: Instructions on Zashi app release process.
+title: 'Release: '
+labels: 'release'
+assignees: ''
+
+---
+### Details about the release
+<!-- Describe any details related to this app release. -->
+
+### Release steps checklist
+This is a brief checklist to ensure all the necessary pre-release and post-release steps have been done:
+
+#### 1. Google Play release:
+   - [ ] Update [CHANGELOG.md](../../CHANGELOG.md)
+   - [ ] Update all supported What's New release notes for Google Play:
+     - [WHATS_NEW_EN.md](../../docs/whatsNew/WHATS_NEW_EN.md)
+     - [WHATS_NEW_ES.md](../../docs/whatsNew/WHATS_NEW_ES.md)
+   - [ ] Create release notes for all supported languages for F-Droid. Note the new file name MUST be equal to the app 
+     `versionCode`.
+     - [EN changelog](../../fastlane/metadata/android/en-US/changelogs/)
+     - [ES changelog](../../fastlane/metadata/android/es/changelogs/)
+   - [ ] Update the build version name **ZCASH_VERSION_NAME** in [gradle.properties](../../gradle.properties) (if required)
+   - [ ] Check if the latest [**Zcash SDK**](https://repo.maven.apache.org/maven2/cash/z/ecc/android/zcash-android-sdk/) with the updated checkpoints is used
+   - [ ] Check if the latest [**Bip39**](https://repo.maven.apache.org/maven2/cash/z/ecc/android/kotlin-bip39/) is used
+   - [ ] Check if the latest app dependencies are used, use `./gradlew dependencyUpdate`
```

Comment:
I'd suggest updating these in a separate PR prior to the release. If they're updated in the release PR then there will never be time to review them. Granted it's hard to review them anyway, but in the corresponding librustzcash PRs I would occasionally find problematic updates.

Reply:
Yes, definitely, we do these in a separate PR. I think about the template as a list of possible steps to check, not necessarily blocking sub-tasks allowing its merge. But I'm in favor of removing it from the list. The dependency update would be a separate recurring issue, which is the perfect way. Currently we use a recuring calendar tasks and dedicated slack channel reportig a new versions of depedndnecies we use.